src/NzbDrone.Core/Notifications/SeriesDeleteMessage.cs

```diff
@@ -0,0 +1,18 @@
+﻿using NzbDrone.Core.MediaFiles;
+using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.Notifications
+{
+    public class SeriesDeleteMessage
+    {
+        public string Message { get; set; }
+        public Series Series { get; set; }
+
+        public bool DeleteFiles { get; set; }
+
+        public override string ToString()
+        {
+            return Message;
+        }
+    }
+}
```

Comment:
Nit: new line at end of file

Reply:
Resolved in upcoming commit

Reply:
same issue as above - VS has a space at the end of the file but it's being stripped